WordPressKitTests/BlazeServiceRemoteTests.swift

```diff
@@ -77,4 +76,38 @@ final class BlazeServiceRemoteTests: RemoteTestCase, RESTTestable {
         encoder.outputFormatting = [.sortedKeys, .prettyPrinted]
         return try encoder.encode(object)
     }
+
+    // MARK: - Campaigns
+
+    func testDecodeCampaignsSearchResponse() throws {
+        // Given
+        let url = try XCTUnwrap(Bundle(for: BlazeServiceRemoteTests.self).url(forResource: "blaze-campaigns-search", withExtension: "json"))
+        let data = try Data(contentsOf: url)
```

Comment:
FYI there's this method: `stubRemoteResponse(_ endpoint: String, filename: String, contentType: ResponseContentType, status: Int32 = 200)`

Also, wdyt of calling BlazeServiceRemote.searchCampaigns(forSiteId: page: callback:) in the test?

Reply:
Updated. I also covered failure scenarios.